packages/ui/src/components/viewport/StageView.tsx

```diff
@@ -17,7 +17,7 @@ export function StageView({
   ...rest
 }: StageViewProps) {
   const localRef = useRef<HTMLDivElement>();
-  const {background} = useSharedSettings();
+  const {background} = usePreviewSettings();
```

Comment:
`StageView` is used to display the animation in all three places:
- `PresentationMode` - uses rendering settings
- `RenderingViewport` - uses rendering settings
- `PreviewStage` - uses preview settings

So the background should probably be passed here as a prop depending on the context in which the stage is used.